pages/common/sshfs.md

```diff
@@ -0,0 +1,19 @@
+# SSHFS
+
+> filesystem client based on ssh
+
+- mounting remote directory
```

Comment:
Descriptions of a command ideally should start with a capital letter, and end in a colon. Perhaps travis-ci crashed - it should have warned about this :P